kfdef/source/master/kfctl_aws_cognito.yaml

```diff
@@ -339,12 +331,12 @@ spec:
       overlays:
       - application
       parameters:
+      - name: namespace
```

Comment:
Shouldn't this be 'istio-namespace' instead of 'namespace' since the line:
https://github.com/kubeflow/manifests/pull/944/files#diff-e7eae0bee4eed3f89046e8546a08bce1R24

Refers to this as 'istio-namespace'?

Am I missing something?

Reply:
`namespace` is used to override customize resource files namespace. I notice kfctl will by default use `kubeflow` even I have fix value `istio-system` in the manifest.https://github.com/kubeflow/manifests/blob/a235b9825390da5c8b7cf8f479c5802509d4e936/aws/aws-istio-authz-adaptor/base/kustomization.yaml#L3

It still has `istio-namespace` https://github.com/kubeflow/manifests/blob/a235b9825390da5c8b7cf8f479c5802509d4e936/aws/aws-istio-authz-adaptor/base/params.env#L3  and I just leave a default value and don't explicitly override it in the manifest.

Reply:
Ok thanks.